### PR TITLE
Separate import and export providers in select field

### PR DESCRIPTION
**Pull request by MiragePresent**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #238 

Providers injections into import/export form types instead of using the common custom select

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
I'd much rather create a ImportProviderChoiceType and a ExportProviderChoiceType, can you change your PR?

**Comment by MiragePresent** [comment]
Yep

File: src/DataDefinitionsBundle/Form/Type/ImportProviderChoiceType.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+
```
Comment (dpfaffenbauer) [inline src/DataDefinitionsBundle/Form/Type/ImportProviderChoiceType.php]: Copy right headers

File: src/DataDefinitionsBundle/Form/Type/ExportProviderChoiceType.php [hunk 2]
```diff
@@ -1,32 +1,22 @@
 <?php
-/**
```
Comment (dpfaffenbauer) [inline src/DataDefinitionsBundle/Form/Type/ExportProviderChoiceType.php]: why'd you remove the copyright headers?

File: src/DataDefinitionsBundle/Form/Type/ImportProviderChoiceType.php (hunk 1 above)
Comment (MiragePresent) [reply src/DataDefinitionsBundle/Form/Type/ImportProviderChoiceType.php]: fixed

File: src/DataDefinitionsBundle/Form/Type/ExportProviderChoiceType.php (hunk 2 above)
Comment (MiragePresent) [reply src/DataDefinitionsBundle/Form/Type/ExportProviderChoiceType.php]: I deleted `ProviderChoiceType` and created `ImportProviderChoiceType` without copyright, but git shows it like one file. It's already fixed

**Comment by dpfaffenbauer** [comment]
thanks

**Pull request merged**